config/jobs/kubernetes/sig-scalability/sig-scalability-golang.yaml

```diff
@@ -38,7 +38,9 @@ periodics:
     preset-e2e-kubemark-common: "true"
     preset-e2e-kubemark-gce-scale: "true"
   annotations:
+    testgrid-alert-email: go-kubernetes-scalability-tickets@googlegroups.com
     testgrid-dashboards: sig-scalability-golang
+    testgrid-num-failures-to-alert: "2"
```

Comment:
Question, are quotes around 2 necessary?

Reply:
I grepped `testgrid-num-failures-to-alert` over whole `k8s.io/test-infra/config` directory and each occurence had quotes.